src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/XPlatUtility.cs

```diff
@@ -44,21 +44,5 @@ public static void SetUserAgent()
             UserAgent.SetUserAgentString(new UserAgentStringBuilder("NuGet xplat"));
```

Comment:
@mishra14 please open an issue to track automating this process as part of build.ps1/build.sh to avoid noise in PRs. Thanks.